Cargo.toml

```diff
@@ -23,8 +23,8 @@ members = [
 	"test-utils",
 	"types",
 	"rpc/json-rpc",
-	"rpc/rpc-types",
-	"rpc/rpc-interface",
+	"rpc/types",
+	"rpc/interface",
```

Comment:
Sort of a grey area leftover from https://github.com/Cuprate/cuprate/pull/164.

The crate is `cuprate-rpc-types`, but the directory is `rpc/types`. Should it technically be `rpc/rpc-types`?

Maybe there's leeway here if the directory already has the name, e.g. `cuprate-consensus-rules` is `consensus/rules`.